frontend/app/src/store/history/transactions.ts

```diff
@@ -1,9 +1,12 @@
 import isEqual from 'lodash/isEqual';
 import { type Ref } from 'vue';
+import { useStatusUpdater } from '@/composables/status';
```

Comment:
this should now be optional, if the method is not already in auto-imports then a build should add it